### PR TITLE
Add Wishlist

### PR DESCRIPTION
**Pull request by gadgetfather**
I have created a wishlist page. Please review it.
 
image
image

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    padding-bottom: 6px;
+    border-bottom: 1px solid rgb(207, 207, 207);
+}
+.brand-name{
+    font-size: var(--fs3);
+}
+/* searchbar */
+.search-bar{
+    margin-left: 10px;
+    margin-right: 10px;
```
Comment (starver20) [inline wishlist/wishlist.css:22]: Can be replaced with margin:0 10px;

**Review by starver20 — APPROVED** [review]

File: wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+}
+.navbar{
+    margin-bottom: 5px;
+    padding: 1.3rem var(--spacing3) ;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 6px;
+    border-bottom: 1px solid rgb(207, 207, 207);
+}
```
Comment (ruchiragoswami) [inline wishlist/wishlist.css:15]: Suggestion to add one empty line at the end of the code block, for readability.

File: wishlist/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    background-color: white;
+    border: 1px solid var(--primary-color);
+}
+
+.search-bar i{
+    margin-left: 4px;
+    color: gray;
+}
+.search-bar input{
+    padding: 10px 10px;
```
Comment (ruchiragoswami) [inline wishlist/wishlist.css:34]: this is same as 
padding : 10px; 
:)

File: wishlist/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    margin-bottom: 0.8rem;
+    margin-right: 9.75px;
+    margin-left: 9.75px;
+    padding-bottom: 0.5rem;
+}
+
+.wishlist-section .product-card .btn{
+    font-size: 1rem;
+}
+
```
Comment (ruchiragoswami) [inline wishlist/wishlist.css:101]: I personally like to remove all empty lines in-between the code blocks. I keep only one line in between for readability .
If there is a reason, I keep more empty lines.

**Review by ruchiragoswami — APPROVED** [review]
Great work Akshay ! 
I am approving with comments :)

File: wishlist/wishlist.css (hunk 3 above)
Comment (gadgetfather) [reply wishlist/wishlist.css:34]: yea I will do that

File: wishlist/wishlist.css (hunk 2 above)
Comment (gadgetfather) [reply wishlist/wishlist.css:15]: Sure

File: wishlist/wishlist.css (hunk 1 above)
Comment (gadgetfather) [reply wishlist/wishlist.css:22]: oh yes

**Pull request merged**